<launch>
    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch">
        <arg name="port" default="9090"/>
    </include>

    <node pkg="talk" type="talk_node.py" name="talk_node" output="screen">
        <param name="language" value="fr"/> <!-- For french use "fr" and for english use "en" -->
        <param name="mouth_signal_gain" value="0.1"/>
        <param name="frame_sample_count" value="256"/>

        <remap from="audio_out" to="audio_float_22050"/>
    </node>
</launch>
